.github/workflows/node.js.yml

```diff
@@ -10,6 +10,34 @@ on:
         branches: [master, feature/*, mynah-dev]
 
 jobs:
+    check-test-files:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
+            - name: Check for the presence test files
+              id: check_file
+              run: |
+                  testFiles=$(git diff --name-only ${{ github.event.pull_request.base.sha }} | { grep '\.test\.ts$' || true; })
```

Comment:
I wonder if we can make this smarter? E.g. if source files modified + no test files modified. Getting this for package.json changes or other infra changes would get old quickly I think, but maybe a worthy tradeoff if its not possible.

Reply:
> if source files modified + no test files modified

:+1: It should behave like that now